### PR TITLE
V0 Skirt Collection

### PR DESCRIPTION
**Pull request by harry-boe**
This is a mod with different Skirts as described in the readme.
I followed the conventions below

Harry

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ like so:
 | | [Voron2.4 Y Endstop Relocation](./hartk1213/Voron2.4_Y_Endstop_Relocation) | This mod relocates the Y endstop to the top A drive piece | :x:| :x: | :heavy_check_mark: |:x:|:x: |
 | | [Voron2.4 GE5C](./hartk1213/Voron2.4_GE5C) | This mod replaces the stack of printed z joints with a metal spherical bearing  | :x:| :x: | :heavy_check_mark: |:x:|:x:|
 | | [Voron2.4 SexBolt ZEndstop](./hartk1213/Voron2.4_SexBolt_ZEndstop) | This mod replaces stock V2 Z endstop with a different style| :x:| :x: | :heavy_check_mark: |:x:|:heavy_check_mark: |
+| hbo | [Voron0 skirs](./hbo/Voron0_Skirts) | Collection of Voron 0 Skirts | :heavy_check_mark:| :x: | :x: |:x:|
```
Comment (MapleLeafMakers) [inline printer_mods/README.md:189]: typo in `skirs`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Collection of Voron 0 Skirts
+
+
+
+
+This is a Collection of Voron 0 skirts that provide some additional features and bring the asthetics closer to the bigger VORON brothers.
+
+![Printed Panel](Images/VORON_V0.1_2022.png)
+
+Right now there are 3 variations for the and one commmon rear.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:10]: "3 variations for the...." I think you missed a word here.

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+To start with the rear ..
+
+## Rear Skirts
+
+The rear skirts match all the different front skirts and have the following features
+
+- integrated power inlet mount. To be used with the "standard" v0 power inlet
+- 2 keystone mount to get acces to the raspery ports
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:18]: `acces` -> `access`
`raspery` -> `raspberry pi`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- bowden clip mount support. Disigned for the E3D style bowden clips. This allows to push the fillament and not the tube from the outside. Read for the rabit feader :)
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:19]: `Disigned` -> `Designed`
`fillament` -> `filament`
`rabit feader` -> `rabbit feeder`  (Consider calling this by it's full name "Enraged Rabbit Carrot Feeder", or just ERCF.

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+
+![Printed Panel](Images/VORON_V0.1_Rear.png)
+
+## Headless
+
+This version is intendet for users that like to run the V0 in headless mode i.e. without display
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:26]: `intendet` -> `intended`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+
+![Printed Panel](Images/VORON_V0.1_Headless.png)
+
+## Mini Display
+
+This version is desiged for the 12864 mini display (same as the V2.4 display)
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:33]: `12864 mini` -> `mini 12864`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+The design includes a little part to get acces to the reset switsch.
+
+![Printed Panel](Images/VORON_V0.1_MiniDisplay.png)
+
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:40]: `desiged` -> `designed`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+This version is intendet for users that like to run the V0 in headless mode i.e. without display
+
+
+![Printed Panel](Images/VORON_V0.1_Headless.png)
+
+## Mini Display
+
+This version is desiged for the 12864 mini display (same as the V2.4 display)
+The design includes a little part to get acces to the reset switsch.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:34]: `acces` -> `access`
`switsch` -> `switch`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+![Printed Panel](Images/VORON_V0.1_MiniDisplay.png)
+
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
+The mounts are desihed to bolt on the frame and display with original screws from the back.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:41]: `desihed` -> `designed`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+Therfore similar 4.3 inch display might not work without modifications
+I'm using a 60cm DSI cable to connect the diplay with a Raspery back on the back of the V0
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:43]: `Raspery` -> `Raspberry Pi`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
+The mounts are desihed to bolt on the frame and display with original screws from the back.
+Therfore similar 4.3 inch display might not work without modifications
+I'm using a 60cm DSI cable to connect the diplay with a Raspery back on the back of the V0 
+(a Raspberry Pico mount mod is in preparation)
+
+### Reset / Emergency switch
+The fron display mount also provide a mounting option for a reset or inmergency switch.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:47]: `fron` -> `front`
`inmergency` -> `emergency`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+I'm using this switch to reset a BTT PSU relays. 
+
+The switch in the cad is a standard 7mm momentray switch that is easy to source even from local providers.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:50]: `momentray` -> `momentary`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+Make shire to get a Normal OPEN switch if you plan to use it with the BTT PSU Relays.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:51]: `shire` -> `sure`
`Normal OPEN` -> `Normally Open (NO)`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+(A Relays mount and firmware configs are in preparation)
+
+![Printed Panel](Images/VORON_V0.1_Waveshare.png)
+
+## Printing
+
+All parts print with standard Voron print setting without any supports.
+All parts fit on a V0. However the max out the build volume on the z axis.
+If you need to print on a V0 make shure that you bed springs are really stiff by adding more tention with washers etc.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:60]: `shure` -> `sure`
`you` -> `your`
`tention` -> `tension`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+I had some problems getting nice parts out of the V0 due to the nozzle mooving he part dur to flex on the bed springs.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:61]: `mooving` -> `moving`
`he` -> `the`
`dur` -> `due`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+## BOM Notes
+
+Obviously depending on which mod you are going to use ..
+
+Waveshare Display:  https://www.waveshare.com/4.3inch-dsi-lcd.htm
+
+12864 Mini Display:  https://biqu.equipment/collections/lcd/products/bigtreetech-mini12864-v1-0-lcd-screen-rgb-backlight-mini-display-supports-marlin-diy-for-skr-3d-printer-part
+
+Rubber Feets: Ther are many options. The one in the CAD are this ones here https://de.aliexpress.com/item/1005004478863418.html?spm=a2g0o.order_list.0.0.13575c5f8HKvPU&gatewayAdapt=glo2deu
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:72]: `Feets` -> `Feet`
`Ther` -> `There`
`this` -> `these`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+
+7mm Momentary switch: https://www.aliexpress.com/i/32711341102.html?gatewayAdapt=glo2deu
+
+## CAD  Notes
+
+The CAD includes all varients in one file.
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:78]: `varients` -> `variants`

File: printer_mods/hbo/Voron0_Skirts/Readme.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+You need to show/hode the version you like to work with
```
Comment (MapleLeafMakers) [inline printer_mods/hbo/Voron0_Skirts/Readme.md:79]: `hode` -> `hide`

**Review by MapleLeafMakers — COMMENTED** [review]
- File and folder cannot contain spaces
   - printer_mods/hbo/Voron0_Skirts/CAD/AZ Skirts VORON_V0.1.f3z
   - printer_mods/hbo/Voron0_Skirts/CAD/AZ Skirts VORON_V0.1.step.zip

File: printer_mods/README.md (hunk 1 above)
Comment (harry-boe) [reply printer_mods/README.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 2 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 3 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 4 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 5 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 14 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

**Comment by harry-boe** [comment]
As in the commit description.

Removed spaces in  CAD filenames
Typos Fixed in readme's
Cleanup of links to Aliexpress sources (removed not required infos in the url params)
+ Added description for skirt fans
+ Added missing Bottom Plate stl file
+ added file name prefix for agent colors

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 15 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 16 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 17 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

File: printer_mods/hbo/Voron0_Skirts/Readme.md (hunk 18 above)
Comment (harry-boe) [reply printer_mods/hbo/Voron0_Skirts/Readme.md]: fixed

**Comment by andrepl** [comment]
The changes look good, but you've missed a few of the typos in the readme, they're all the ones hidden behind this: 
image

**Comment by harry-boe** [comment]
readme updated and checked with a spell checker

**Comment by MapleLeafMakers** [comment]
just 1 left:
image

**Comment by harry-boe** [comment]
last typos fixed

**Comment by harry-boe** [comment]
Is there anything I need to do to get this merged ?

**Comment by harry-boe** [comment]
it's been a while now. Anything I need to do here ?

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**